CIPs/CIP-19/CIP-19.md

```diff
@@ -0,0 +1,180 @@
+| `image`            | an image                       | IPFS CID                                                     |          | optional |                              |
+| `description`      | a short description            | string                                                       | 420 char | optional | This is my cool description. |
+| `emoji`            | an emoji                       | unicode                                                      | 2 char   | optional | 🔢                            |
+| `background`       | a background image (3:1 ratio) | IPFS CID                                                     |          | optional |                              |
+| `birthDate`        | a date of birth                | [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)           | 10 char  | optional | 1990-04-24                   |
+| `url`              | a url                          | string                                                       | 240 char | optional | http://ceramic.network       |
+| `gender`           | a gender                       | string                                                       | 42 char  | optional | female                       |
+| `residenceCity`    | a city of residence            | string                                                       | 140 char | optional | Berlin                       |
+| `residenceCountry` | a country of residence         | [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) | 2 char   | optional | DE                           |
+| `nationalities`    | nationalities                  | array of [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) values | 40 char  | optional | CN                           |
+| `affiliations`     | affiliations                   | array of strings                                             | 240 char | optional | Ceramic Ecosystem Alliance   |
+
+**Deployment:** `<reference-schema-DocID>`
+
+```json
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "BasicProfile",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "maxLength": 150
+    },
+    "image": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string",
+      "maxLength": 420
+    },
+    "emoji": {
+      "type": "string",
+      "maxLength": 2
+    },
+    "background": {
+      "type": "string"
+    },
+    "birthDate": {
+      "type": "string",
+      "format": "date"
+    },
+    "url": {
+      "type": "string",
+      "maxLength": 240
+    },
+    "gender": {
+      "type": "string",
+      "maxLength": 42
+    },
+    "homeLocation": {
+      "type": "string",
+      "maxLength": 140
+    },
+    "residenceCountry": {
+      "type": "string",
+      "pattern": "^[A-Z]{2}$"
+    },
+    "nationalities": {
+      "oneOf": [
```

Comment:
Is think we should make it so it's always an array, rather than having to check at runtime for an array or scalar.

Reply:
Good point!